### ISSUE TITLE
Missing typings in 1.7.2

### ISSUE THREAD

**Issue by josephluck** [berstend/service-worker-router#5]
It seems that there's no typings published to npm in 1.7.2 😢

**Comment by Lngramos** [comment]
It appears a fix for this has already been merged in (https://github.com/berstend/service-worker-router/pull/3), but not yet released.

@berstend any chance you have some time to create a new release that includes the merged fix?

Thanks in advance 🙏

**Comment by hellodword** [comment]
> It appears a fix for this has already been merged in (#3), but not yet released.
> 
> @berstend any chance you have some time to create a new release that includes the merged fix?
> 
> Thanks in advance 🙏

got the same issue

not want to fork

so i used it with `git submodule`

```sh
cd app
mkdir modules

git submodule add https://github.com/berstend/service-worker-router modules/service-worker-router

# install it as a local module
npm install ./modules/service-worker-router

```


hope helpful

**Comment by berstend** [comment]
Apologies @josephluck @Lngramos @tadeuszwojcik - I just published `1.7.4` on npm with the typings fix as well as fresh dependencies :)
